packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -23,20 +23,25 @@ export interface CachePermissionOptions {
   cache?: CacheSelector;
 }
 
-export class CachePermission {
-  cacheRole: CacheRole;
+export interface CachePermission {
+  role: CacheRole;
   cache: CacheSelector;
-
-  constructor(cacheRole: CacheRole, options?: CachePermissionOptions) {
-    this.cacheRole = cacheRole;
-    this.cache = options?.cache ?? AllCaches;
-  }
 }
 
+// export class CachePermission {
+//   cacheRole: CacheRole;
+//   cache: CacheSelector;
+//
+//   constructor(cacheRole: CacheRole, options?: CachePermissionOptions) {
+//     this.cacheRole = cacheRole;
+//     this.cache = options?.cache ?? AllCaches;
+//   }
+// }
+
 export enum TopicRole {
   None = 'none',
-  ReadWrite = 'readwrite',
-  ReadOnly = 'readonly',
+  PublishSubscribe = 'readwrite',
```

Comment:
Do we want to change the enum values as well for consistency?

Reply:
+1 from me --a developer may print a enum value be surprised by the inconsistency.

Reply:
Fixed in https://github.com/momentohq/client-sdk-javascript/pull/598/commits/649148592f9a49d01715fa7258ff2bee2c8e741d